decidim-meetings/app/views/decidim/meetings/meetings/show.html.erb

```diff
@@ -33,7 +33,13 @@ edit_link(
     <div class="card extra">
       <div class="card__content">
         <div class="extra__date">
-          <%= l meeting.start_time, format: "%d" %> <span class="extra__month"><%= l meeting.start_time, format: "%B" %></span>
+          <%= l meeting.start_time, format: "%d" %>
+          <span class="extra__month"><%= l meeting.start_time, format: "%B" %>
+            <% year = l meeting.start_time, format: "%Y" %>
+            <% if year.to_i < Date.today.year %>
+              <%= l meeting.start_time, format: "%Y" %>
+            <% end %>
```

Comment:
```suggestion
              <%= l(meeting.start_time, format: "%Y") if meeting.start_time.year != Date.current.year %>
```
I suggest this change which covers dates in the future as well. It is also a little bit more consice